server/server/views/recover.py

```diff
@@ -0,0 +1,59 @@
+import transaction
+from cornice import Service
+from passlib.totp import generate_secret
+from pyramid_mailer.mailer import Mailer
+from pyramid_mailer.message import Message
+from passlib.hash import pbkdf2_sha256
+from passlib.totp import generate_secret
```

Comment:
It's already imported in line 3. Also I see here many unused imports, review them and remove extra ones.